Tasks/GradleV2/task.json

```diff
@@ -16,7 +16,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 181,
+        "Minor": 185,
```

Comment:
Should there be update for task.loc.json file also?

Reply:
@anatolybolshakov Yes, updated.